recipes/python-glmnet/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "v2.0.0" %}
+
+package:
+  name: glmnet
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/civisanalytics/python-glmnet.git
+  git_rev: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - setuptools
+    - numpy x.x
+    - scikit-learn >=0.18.0
+    - scipy >=0.14.1
+    - gcc  # [linux,osx]
+
+  run:
+    - numpy x.x
+    - scikit-learn >=0.18.0
+    - scipy >=0.14.1
+    - libgcc  # [linux]
+    - libgfortran  # [osx]
+
+test:
+  imports:
+    - glmnet
+
+about:
+  home: https://github.com/civisanalytics/python-glmnet
+  license: GPL 2
+  summary: 'Python wrapper for glmnet'
```

Comment:
It looks like there is a license file, so please add:

```
license_file: LICENSE
```

Also please add `dev_url: https://github.com/civisanalytics/python-glmnet`